src/pytasks.py

```diff
@@ -0,0 +1,31 @@
+""" Function from  previous tasks """
+
+
+def fizzbuzz(range_=2):
+    number_list = []
+    for i in range(1, range_ + 1):
+        if i % 3 == 0 or i % 5 == 0:
+            number_list.append("Fizz" * (i % 3 == 0) + "Buzz" * (i % 5 == 0))
+        else:
+            number_list.append(i)
+    return number_list
+
+
+def gen_numbers(numbers=3):
+    return {digit: digit ** 2 for digit in range(1, numbers + 1)}
+
+
+def palindrome(num=11):
```

Comment:
why this number should be default?  we should test our numbers, not truncated or reworked ones. it's about all these funcs.